inscriptions/ISic000560.xml

```diff
@@ -191,56 +196,22 @@
-                        <ex>anibus</ex>
-                    </expan>
-                    .<lb n="2"/>Fortis
-                    <expan>
-                        <abbr>Caesar</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ser</abbr>
-                        <ex>vus</ex>
-                    </expan>
-                    <lb n="3"/>sibi et Charidi<lb n="4"/>coniugi bene<lb n="5"/>merenti et
-                    <expan>
-                        <abbr>P</abbr>
-                        <ex>ublio</ex>
-                    </expan>
-                    Aelio
-                    <lb n="6"/>
-                    <expan>
-                        <abbr>Aug</abbr>
-                        <ex>usti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>lib</abbr>
-                        <ex>erto</ex>
-                    </expan>
-                    Felici poste
-                    <lb n="7" break="no"/>
-                    <expan>
-                        <abbr>risq</abbr>
-                        <ex>ue</ex>
-                    </expan>
-                    suorum fecit.
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/>fortis <g ref="#interpunct">·</g> <expan><abbr>Caesar</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>ser</abbr><ex>vus</ex></expan>
+                    <lb n="3"/>sibi <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> Charidi
+                    <lb n="4"/>coniugi <g ref="#interpunct">·</g> bene
+                    <lb n="5"/>merenti <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> <expan><abbr>P</abbr><ex>ublio</ex></expan> <g ref="#interpunct">·</g> Aelio
+                    <lb n="6"/><expan><abbr>Aug</abbr><ex>usti</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>lib</abbr><ex>erto</ex></expan> <g ref="#interpunct">·</g> felici <g ref="#interpunct">·</g> poste
+                    <lb n="7" break="no"/><expan><abbr>risq</abbr><ex>ue</ex></expan> <g ref="#interpunct">·</g> suorum <g ref="#interpunct">·</g> fecit
                 </ab>
-                <!--This EDR text 138337 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#NCash"> 
+                <p>To the spirits of the Underworld. A slave of the mighty emperor made (this monument) for himself and his well-deserving wife Charis and for Publius Aelius, a fortunate freedman of the emperor, (and) for his own descendants</p>
```

Comment:
To the spirits of the dead. Fortis, slave of Caesar... Publius Aelius Felix, freedman of Augustus,